packages/vite-utils/tsconfig.build.json

```diff
@@ -1,10 +1,5 @@
 {
   "extends": "@eternagame/tsconfig/tsconfig.node.json",
-  "compilerOptions": {
```

Comment:
We get rid of path aliasing because the vite-utils package never uses it?

Reply:
Not just never uses it - can't use it. When we run `vite build` for vite-utils, vite internally compiles the vite.config.ts. Vite/esbuild doesn't support tsconfig path aliases - that's something we configure in the vite config itself. But on first build, the compilation of vite.config.ts to build vite-utils (while the config itself uses vite-utils!) will not be using our vite config, and so will not have any aliases registered.